src/ExampleScene.gd

```diff
@@ -90,87 +150,168 @@ func showcase_levels() -> void:
 	Log.err("Log.err()")
 	Log.error("Log.error()")
 
+
+## Showcase the custom color functionality of [code]Log.to_pretty()[/code].
 func showcase_colors() -> void:
-	print_header("Custom Colors")
-	Log.pr("custom colors")
-	print_rich(Log.to_pretty(1))
-	print_rich(Log.to_pretty(1, {color_scheme={TYPE_INT: "purple"}}))
+	print_header("Custom Colors with [code]Log.to_pretty()[/code]")
+
+	print("Plain ol' print()")
+	print_rich(Log.to_pretty(
+			"Log.gd standard colors with [code]Log.to_pretty()[/code]",
+			{"color": custom_color}))
+	print_rich(Log.to_pretty(
+			"Custom colors with [code]Log.to_pretty()[/code]",
+			{"color": custom_color}))
+	print_rich(
+			Log.to_pretty("Disable colors with [code]Log.to_pretty()[/code]",
+			{"disable_colors": true}))
 
-	Log.pr("disabled colors")
-	print_rich(Log.to_pretty(1, {disable_colors=true}))
 
+## Showcase ints and floats
 func showcase_ints_and_floats() -> void:
 	print_header("Ints and Floats")
+
 	print(42, 3.14)
-	Log.pr(42, 3.14)
+	Log.log(42, 3.14)
+	Log.blank()
 
 	print(1)
-	Log.pr(1)
+	Log.log(1)
+	Log.blank()
 
 	print(1.0)
-	Log.pr(1.0)
+	Log.log(1.0)
+
 
+## Showcase the printing of Vectors, float and int
 func showcase_vectors() -> void:
 	print_header("Vectors")
+
 	print(Vector2())
-	Log.pr(Vector2())
+	Log.log(Vector2())
+	Log.blank()
 
 	print(Vector3(2.3, 1.4, 0.5))
-	Log.pr(Vector3(2.3, 1.4, 0.5))
+	Log.log(Vector3(2.3, 1.4, 0.5))
+	Log.blank()
 
 	print(Vector2i.LEFT)
-	Log.pr(Vector2i.LEFT)
+	Log.log(Vector2i.LEFT)
+	Log.blank()
 
 	print(Vector3i.UP)
-	Log.pr(Vector3i.UP)
+	Log.log(Vector3i.UP)
+
 
+## Showcase strings and string names
 func showcase_strings() -> void:
 	print_header("Strings and String Names")
-	Log.pr("Hello", &"World")
+
 	print("Hello, World!")
-	Log.pr("Hello, World!")
+	Log.log("Hello, World!")
+	Log.blank()
+
+	print("Hello", &"World")
+	Log.log("Hello", &"World")
+	Log.blank()
+
 	print(&"Hi there!")
-	Log.pr(&"Hi there!")
+	Log.log(&"Hi there!")
+
 
+## Showcase mixed arrays of various nesting levels
 func showcase_arrays() -> void:
 	print_header("Arrays")
-	print([1, 2.0, 3, 4.0])
-	Log.pr([1, 2.0, 3, 4.0])
-	Log.pr(1, 2.0, [3, 4.0])
-	Log.prn([1, 2.0, 3, 4.0])
 
-	Log.pr("an array of vectors", [
+	var simple_array: Array = [1, 2.0, 3, 4.0]
+	print(simple_array)
+	Log.log(simple_array)
+	Log.prn(simple_array)
+	Log.blank()
+
+	# Until Godot allows unpacking of arrays, I'm not exactly sure how to move
+	# this print statement into a variable effectively.
```

Comment:
You can actually do this with `callv` like:

```
	var mixed_args = [1, 2.0, [3, 4.0]]
	print.callv(mixed_args)
	Log.log.callv(mixed_args)
	Log.prn.callv(mixed_args)
```

Reply:
the trade off to me is whether you show the typical usage or not - i think it's fine either way